docs/docs/api-reference/starlark-reference/directory.md

```diff
@@ -5,24 +5,32 @@ sidebar_label: Directory
 
 The `Directory` constructor creates a `Directory` object that represents a directory inside an existing service (see
 the [ServiceConfig][service-config] object).
-Directory object can be either a files artifact or a persistent directory, depending on the arguments passed to the 
+Directory object can be either one or many files artifacts or a persistent directory, depending on the arguments passed to the 
 constructor.
 
 ```python
+# composed with only one file artifact:
 file_artifact_directory = Directory(
-    artifact_name=files_artifact_1,
+    artifact_names=files_artifact_1,
```

Comment:
will have to edit this when the syntax gets changed

Reply:
updated, thanks for the reminder